sdk/firstload/src/browserExtension/extensionListener.ts

```diff
@@ -1,4 +1,4 @@
-import { H } from '..'
+import { H } from '../index'
```

Comment:
Similar to https://github.com/highlight/highlight/pull/5089/commits/fd58dbaa5938a9ee77190946ddeafe8887fad2ce#r1174279883, `..` is also not a valid relative specifier.